Packs/OpenAI/Integrations/OpenAIGPT/README.md

```diff
@@ -0,0 +1,94 @@
+
+## OpenAI GPT
+### Instance Configuration
+
+- #### Generate an API Key
+    1. Sign-up or login to [OpenAI developer platform](https://platform.openai.com).
+    2. Generate a new API Key at [OpenAI developer platform - api-keys](https://platform.openai.com/api-keys).
+
+- #### Choose a GPT model to interact with
+    1. The integration utilizes the **'Chat Completions'** endpoint merely. Therefore, it will only be possible to configure models that support this endpoint (_https://api.openai.com/v1/chat/completions_). 
```

Comment:
```suggestion
    1. The integration utilizes the **Chat Completions** endpoint. Therefore, it will only be possible to configure models that support this endpoint (_https://api.openai.com/v1/chat/completions_). 
```